spread/build/ubuntu/task.yaml

```diff
@@ -1,5 +1,5 @@
 summary: Build (on Ubuntu)
-systems: [-fedora-*]
+systems: [-fedora-*, -alpine-*]
```

Comment:
Should this maybe be changed to something like `systems: [ubuntu-*]` ?

Reply:
Yes, probably :)